VideoSampleCaptureRender/ViewController.swift

```diff
@@ -219,7 +219,11 @@ class ViewController: UIViewController, UITextFieldDelegate {
         
         960x540 video will fill modern iPhone screens. However, older devices (< iPhone 5s, iPad Air 2) will have trouble rendering this quality. */
         
-        return TWCVideoConstraints(maxSize: TWCVideoConstraintsSize960x540, minSize: TWCVideoConstraintsSize960x540, maxFrameRate: 0, minFrameRate: 0)
+        if (Platform.isLowPerformanceDevice) {
+            return TWCVideoConstraints(maxSize: TWCVideoConstraintsSize640x480, minSize: TWCVideoConstraintsSize640x480, maxFrameRate: 0, minFrameRate: 0)
+        } else {
+            return TWCVideoConstraints(maxSize: TWCVideoConstraintsSize960x540, minSize: TWCVideoConstraintsSize960x540, maxFrameRate: 0, minFrameRate: 0)
+        }
```

Comment:
If there will only be two performance "bins", then you should reduce the low performance constraints. An A5 class device is going to struggle with 640x480x30 which is what these constraints will resolve to. I would suggest using either:

640x480x15
480x360x15

For the low performance devices.